astropy/units/function/logarithmic.py

```diff
@@ -216,12 +216,12 @@ class LogQuantity(FunctionQuantity):
     subclasses, as in::
 
         >>> import astropy.units as u
-        >>> u.Magnitude(15.)
-        <Magnitude 15.0 mag>
-        >>> u.Magnitude(10.*u.count/u.second)
+        >>> u.Magnitude(12.5)  # doctest: +FLOAT_CMP
```

Comment:
Are all these ``FLOAT_CMP`` necessary? The float format usually isn't messed up when no arithmetic is done, isn't it?